extensions/amp-list/0.1/test/test-amp-list.js

```diff
@@ -895,6 +895,32 @@ describes.repeated(
             expect(element.getAttribute('src')).to.equal('');
           });
 
+          it('should not render if [src] has changed since the fetch was initiated', async () => {
+            const foo = doc.createElement('div');
+            const bar = doc.createElement('div');
+
+            const firstPromise = Promise.resolve({items: [foo]});
+            const secondPromise = firstPromise.then(() => ({items: [bar]}));
+
+            listMock
+              .expects('fetch_')
+              .returns(firstPromise)
+              .returns(secondPromise)
```

Comment:
Does this `returns()` make the line above noop? 

You might need `onCall(n).returns(m)`: https://sinonjs.org/releases/v7.5.0/stubs/

Reply:
You are right! Can't just assume that things that work in [truth](https://github.com/google/truth) also work here :). 

I'm guessing this test still worked anyway since the code didn't have any optimizations to bail even though the result was the same (at least not as high up as i'm testing)

Reply:
Done